source/Halibut.OctopusSample/Halibut.OctopusSample.csproj

```diff
@@ -45,4 +53,8 @@
     </Compile>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)'=='netcoreapp2.2'">
```

Comment:
Couldn't see a 'neat' way to skip compilation, so I stubbed out the executable instead.